src/Network/GRPC/MQTT/Message/Packet.hs

```diff
@@ -208,11 +246,16 @@ maxPacketSize = 256 * 2 ^ (20 :: Int) - 128 * 2 ^ (10 :: Int)
 -- @since 0.1.0.0
 makePacketSender ::
   MonadUnliftIO m =>
-  -- | The maximum packet payload size. This must be a 'Word32' value greater 
+  -- | The maximum packet payload size. This must be a 'Word32' value greater
   -- than or equal to 'minPacketSize' and less than 'maxPacketSize'. Otherwise,
   -- a 'PacketSizeError' exception will be thrown.
   Word32 ->
+  -- | The function to publish serialized packets. In practice, this function is 
+  -- always 'publishq' with QoS1. This is only exposed as an argument so that 
+  -- 'makePacketSender' can be given a mock publishing function and tested in 
+  -- the absence of a gRPC service.
```

Comment:
Can you add to this comment an explanation of restrictions on the usage of the `ByteString` argument?